query07.sql

```diff
@@ -3,4 +3,31 @@
 */
 
 -- Enter your SQL query here
-select ...
+
+UPDATE indego_trips_2020_q2
+  SET start_time = replace(start_time,'/','.'),
+      end_time = replace(end_time,'/','.');
+
+CREATE TABLE datetimes AS (
+  SELECT  trip_id
+          , extract(day from to_timestamp(start_time, 'DD.MM.YY HH24:MI')) as startday
+          , extract(day from to_timestamp(end_time, 'DD.MM.YY HH24:MI')) as endday
+          , 2019 as year
+    FROM indego_trips_2019_q2
+  UNION
+  SELECT  trip_id
+          , extract(day from to_timestamp(start_time, 'MM.DD.YY HH24:MI')) as startday
+          , extract(day from to_timestamp(end_time, 'MM.DD.YY HH24:MI')) as endday
+          , 2020 as year
+    FROM indego_trips_2020_q2
+);
+
+SELECT count(*)
+FROM datetimes
+WHERE startday != endday and year=2019
```

Comment:
This actually over-counts. Even though the max trip `duration` is entered as `1440`, some trips start on one day and end _after_ the next day. The solution I used is [here](https://github.com/musa-509-fall-2021/assignment-01-intro-to-geographic-queries/blob/answers/query07.sql#L16-L25) (or a more compact equivalent solution [here](https://github.com/musa-509-fall-2021/assignment-01-intro-to-geographic-queries/blob/answers/query07.sql#L40-L42))